packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -25,6 +25,12 @@ const relevantTableProps: InternalTdsTablePropChange['changed'] = [
  * @slot <default> - <b>Unnamed slot.</b> For the cells.
  * @slot expand-row - Slot for the expanded row.
  */
+
+/**
+ * @part row - Selector for the main row of the table.
+ * @part expand-row - Selector for the expanded row of the table.
+ */
+
```

Comment:
I think this comment should be in the same as the one above for the readme. 

```jsx
/**
 * @slot <default> - <b>Unnamed slot.</b> For the cells.
 * @slot expand-row - Slot for the expanded row.
 * 
 * @part row - Selector for the main row of the table.
 * @part expand-row - Selector for the expanded row of the table.
 */

```

Reply:
Good idea, on it ;)